website/src/pages/SearchPage/components/SearchMacro.tsx

```diff
@@ -130,27 +109,51 @@ export default function SearchMacro({ data, gitRef }: SearchMacroProps) {
   );
 }
 
+<<<<<<< HEAD
+function Row({ title, value, fmt }) {
+  var valFmt = value.center;
+  if (fmt == "time") {
+    valFmt = secondToMicrosecond(value.center);
+  } else if (fmt == "memory") {
+    valFmt = formatByte(value.center);
+  }
+=======
 function Row({ title, value, fmt }:RowProps) {
     var valFmt = value.center
     if (fmt == "time") {
         valFmt = secondToMicrosecond(value.center)
     } else if (fmt == "memory") {
         valFmt = formatByte(value.center)
     }
+>>>>>>> 85cc5af151fab94e3a4fbf022d03bf596ae36e8f
```

Comment:
Something bad has happened with a merge here, the frontend do not start with those.

Reply:
Its running fine on my side 🤔 

![image](https://github.com/vitessio/arewefastyet/assets/124715224/7784f69a-6940-4cbc-b7db-cbe5d42c7c73)

Reply:
It should not, normally. But anyway these merge hints have to be removed